docs/guidance/using-goosehints.md

```diff
@@ -0,0 +1,45 @@
+# Using `.goosehints` in Goose
+
+`.goosehints` are text files used within the Goose environment to provide additional context about your project and improve the communication between the developer and Goose. Ensuring it understands your requirements better and can execute tasks more effectively.
+
+>[!TIP]
+> **Developer toolkit required**
+>
+> To make use of the hints file, you need to have the `developer` toolkit enabled.
+
+This guide will walk you through using your `.goosehints` file to streamline your workflow with custom instructions and context.
+
+## Creating your `.goosehints` file
+You can place a `.goosehints` file in your current working directory or globally at `~/.config/goose/.goosehints`. This file can include any repeated instructions or contextual details relevant to your projects.
+
+A good time to consider adding a `.goosehints` file is when you find your self repeating prompts, or providing the same kind of instructions multiple times.
+
+The `.goosehints` file follows [jinja templating rules][jinja-guide] in case you want to leverage templating to insert file contents or variables. But you can also add instructions in natural language.
+
+### Setting Rules
+
+Some rules you can define for Goose to follow:
```

Comment:
I think I might need to change the lingo here - because these aren't exactly rules. They are prompts/hints/information that Goose includes in all requests - so there can't be an extensive list. I'll rephrase this and re-request a review. @angiejones